src/test/java/org/dinosaur/foodbowl/domain/post/docs/PostControllerDocsTest.java

```diff
@@ -0,0 +1,116 @@
+import java.util.List;
+import org.dinosaur.foodbowl.MockApiTest;
+import org.dinosaur.foodbowl.domain.member.entity.Role.RoleType;
+import org.dinosaur.foodbowl.domain.post.api.PostController;
+import org.dinosaur.foodbowl.domain.post.application.PostService;
+import org.dinosaur.foodbowl.domain.post.dto.response.PostThumbnailResponse;
+import org.dinosaur.foodbowl.global.config.security.jwt.JwtTokenProvider;
+import org.dinosaur.foodbowl.global.dto.PageResponse;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpHeaders;
+import org.springframework.restdocs.headers.HeaderDescriptor;
+import org.springframework.restdocs.payload.FieldDescriptor;
+import org.springframework.restdocs.request.ParameterDescriptor;
+
+@WebMvcTest(PostController.class)
+public class PostControllerDocsTest extends MockApiTest {
+
+    @Autowired
+    private JwtTokenProvider jwtTokenProvider;
+
+    @MockBean
+    private PostService postService;
+
+    @Test
+    @DisplayName("프로필 게시글 목록 조회를 문서화한다.")
+    void findThumbnailsInProfile() throws Exception {
+        PageResponse<PostThumbnailResponse> response = new PageResponse<>(
+                List.of(new PostThumbnailResponse(1L, "path", LocalDateTime.now())),
+                true,
+                true,
+                false,
+                0,
+                1,
+                1,
+                1
+        );
+        given(postService.findThumbnailsInProfile(anyLong(), any(Pageable.class))).willReturn(response);
+
+        var headerDescriptors = new HeaderDescriptor[]{
+                headerWithName(HttpHeaders.AUTHORIZATION).description("서버에서 발급한 엑세스 토큰")
+        };
+
+        var parameterDescriptors = new ParameterDescriptor[]{
+                parameterWithName("memberId").description("프로필 멤버 ID"),
+                parameterWithName("page").optional().description("프로필 게시글 썸네일 목록 페이지 숫자 (입력하지 않으면, default = 0)"),
+                parameterWithName("size").optional().description("프로필 게시글 썸네일 목록 데이터 개수 (입력하지 않으면, default = 18)")
+        };
+
+        var responseFieldDescriptors = new FieldDescriptor[]{
+                fieldWithPath("content").description("게시글 썸네일 정보 목록"),
+                fieldWithPath("content[].postId").description("게시글 ID"),
+                fieldWithPath("content[].thumbnailPath").description("썸네일 경로"),
+                fieldWithPath("content[].createdAt").description("게시글 작성 시간"),
+                fieldWithPath("first").description("첫 페이지 여부"),
+                fieldWithPath("last").description("마지막 페이지 여부"),
+                fieldWithPath("hasNext").description("다음 페이지 존재 여부"),
+                fieldWithPath("currentPage").description("현재 페이지 숫자"),
```

Comment:
```suggestion
                fieldWithPath("currentPage").description("현재 페이지 인덱스"),
```
페이지가 0부터 시작이라 인덱스는 어떤가요??

Reply:
좋습니다 👍